src/webpage.cpp

```diff
@@ -1057,18 +1058,23 @@ QString WebPage::renderBase64(const QByteArray& format)
     return "";
 }
 
-QImage WebPage::renderImage()
+QImage WebPage::renderImage(bool onlyViewport)
```

Comment:
@adeodhar-quark I didn't get a chance to work on it. So still pending ATM. I'll work on it ASAP.